geode-core/src/test/java/org/apache/geode/management/internal/cli/help/HelperIntegrationTest.java

```diff
@@ -34,7 +36,7 @@
   public static void beforeClass() {
     helper = new Helper();
     // use GfshHelpCommand for testing
-    Method[] methods = GfshHelpCommands.class.getMethods();
+    Method[] methods = GfshHelpCommand.class.getMethods();
```

Comment:
Good idea - thank you for your feedback! I don't think the original test ever actually tested the hint command, so I wrote two completely new tests: `HelperIntegrationTest.TestHintWithInput` and `HelperIntegrationTest.TestHintWithoutInput`. PR has also been updated!